zgw_consumers/models.py

```diff
@@ -37,6 +54,13 @@ def __str__(self):
     def save(self, *args, **kwargs):
         if not self.api_root.endswith("/"):
             self.api_root = f"{self.api_root}/"
+
+        if not self.nlx.endswith("/"):
+            self.nlx = f"{self.nlx}/"
+
+        if not self.oas:
+            self.oas = urljoin(self.api_root, "schema/openapi.yaml")
+
         super().save(*args, **kwargs)
 
     def clean(self):
```

Comment:
I'd validate that if `header_key` or `header_value` is set, the other one must also be set.